packages/block-library/src/cover/block.json

```diff
@@ -109,6 +109,9 @@
 			"__experimentalDefaultControls": {
 				"fontSize": true
 			}
+		},
+		"__experimentalLayout": {
+			"allowJustification": false
```

Comment:
It's a bit specific, but we do have a precedent in [allowOrientation](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/layouts/flex.js#L69) which is a valid property that only applies to flex layout (I don't think it's being used in any core blocks right now though). Do you see a better way of selectively disabling controls for a certain layout type?

Reply:
I'm not sure right now, but I guess it would need a new API maybe something like `{ [layout]: allowedControls }`? It would need exploration for sure to find what works and makes sense.

Reply:
I've looked into this a bit and I think this approach is the most consistent with the API we already have in place.
We already have the following properties:

* “allowSwitching": False by default and applicable to all types
* allowEditing": True by default and applicable to all types
* "allowInheriting": True by default and specific to flow/constrained
* "allowSizingOnChildren": False by default and specific to flex
* "allowVerticalAlignment": True by default and specific to flex
* “allowOrientation”: True by default and specific to flex

"allowJustification" (true by default) follows this pattern, and we don't even have to make it specific to "constrained", as "flex" has justification controls too. The same property could be applicable to both types.

Reply:
Thanks for the details here. I guess we can't block this PR, taking into account the above, even if it's not ideal that we have so many flags that are applied to different layouts.